drivers/applications/tla2528_analog_input_demo.cpp

```diff
@@ -0,0 +1,39 @@
+#include "../hardware_map.hpp"
+#include <array>
+#include <libhal-util/bit.hpp>
+#include <libhal-util/i2c.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/input_pin.hpp>
+#include <libhal/units.hpp>
+#include <tla2528.hpp>
+#include <tla2528_adapters.hpp>
+
+using namespace hal::literals;
+using namespace std::chrono_literals;
+
+namespace sjsu::drivers {
+
+void application(application_framework& p_framework)
+{
+
+  hal::byte i2c_address = 0x10;  // 0x10 is the tla address for no resistors
```

Comment:
Why must the user pass this in if this is the default? Why not make the constructor have a default parameter that sets the address to 0x10 and the dev can change this if they need to change it.

Reply:
0x10 is a default parameter. For the demos, I put the address at the top for ease of access, as that way, people aren't wondering how to change the address if they just want to test their TLA and they have it wired to a different address.

Reply:
This is probably best to remove. If the constructor doesn't describe in detail how the i2c address works, it should so anyone reading the header can see exactly when they'd need to use a differnt address.

Reply:
updated documentation & removed address from demos, look it over